doc/article/en-US/i18n-with-aurelia.md

```diff
@@ -268,31 +268,46 @@ the necessary type definitions (d.ts) for the plugins dependencies.
 Here we show how you can do that for i18next and the i18next-xhr-backend, but this
 should be applicable to every other backend choice.
 
-The way to get hold of those is using [typings](https://github.com/typings/typings).
-You can install the typings for [i18next](http://i18next.com/) using this command:
+The way to get hold of those is using using NPM or perhaps Yarn as with other packages.
+In this case the typinhs are at [@types/i18next](https://www.npmjs.com/package/@types/i18next),
+along with the instructions. The command is
 
 ```Shell
-typings install dt~i18next --save --global
+npm install --save @types/i18next
 ```
+or if you are using Yarn
+```Shell
+yarn add @types/i18next
+```
+but in case of Yarn there might be a complication as described at https://github.com/yarnpkg/yarn/issues/4226
+with a working solution.
 
 > Info
 > Alternatively, you can find this file in the plugins repository doc folder: `doc/i18next.d.ts`
 
 As for the XHR-Backend you'll be using:
 
 ```Shell
-typings install dt~i18next-xhr-backend --save --global
+npm install --save @types/i18next-xhr-backend
+```
+or for Yarn
+```Shell
+yarn add @types/i18next-xhr-backend
 ```
 
 > Info
 > Alternative, you can find this file in the plugins repository doc folder: `doc/i18next-xhr-backend.d.ts`
 
 Note: if you decide to use the `doc/*.d.ts` files, you should copy them to another folder, e.g. `custom_typings`.
 
-If you're running a JSPM setup, in order to properly find the `aurelia-i18n.d.ts` file, you can alternatively install it via typings as well.
+If you're running a JSPM setup, in order to properly find the `aurelia-i18n.d.ts` file, you can alternatively install it via npm or yarn as well.
 
 ```Shell
-typings install github:aurelia/i18n
+npm install github:aurelia/i18n
+```
+or for Yarn
+```Shell
+yarn add @types/i18next-xhr-backend
```

Comment:
how does this line fit into the story compared to the given npm install? So why here installing i18next-xhr-backend?

Reply:
A dumb mistake, will fix.